client/src/components/createCredentialDialog/createCredentialDialog.js

```diff
@@ -313,19 +315,19 @@ class CreateCredentialDialog extends React.Component {
         <Form.FormGroup>
           {this.renderFormLabel('Become Method')}
           <Grid.Col sm={7}>
-            <div className="form-split-button">
-              <SplitButton
-                className="form-control"
-                bsStyle="default"
+            <div className="quipucords-dropdownselect">
+              <DropdownSelect
                 title={becomeMethod}
-                id={helpers.generateId('become-method-button')}
+                id="become-method-select"
+                className="form-control"
+                multiselect={false}
               >
                 {this.becomeMethods.map((nextMethod, index) => (
                   <MenuItem key={index} eventKey={`become${index}`} onClick={() => this.setBecomeMethod(nextMethod)}>
```

Comment:
Note: added a class for highlighting the menu items, but you'd have to conditionally display them. Also, I kinda like it without the class: 
` className={{ 'quipucords-dropdownselect-menuitem-selected': ... }}`